app/config.js

```diff
@@ -34,7 +34,7 @@
     loggingURL: "",
     metricsURL: ""
   };
-
+  window.OPENSHIFT_LANG = "zh-CN";
```

Comment:
need comments on it. the OPENSHIFT_LANG is global var, so it default value is en, it not zh-CN default.